legion/engine/physics/diviner/systems/physicssystem.cpp

```diff
@@ -182,7 +182,8 @@ namespace legion::physics
 
     }
 
-    void PhysicsSystem::constructManifoldsWithPrecursors(ecs::component_container<diviner::rigidbody>& rigidbodies, std::vector<byte>& hasRigidBodies, physics_manifold_precursor& precursorA, physics_manifold_precursor& precursorB,
+    void PhysicsSystem::constructManifoldsWithPrecursors(ecs::component_container<diviner::rigidbody>& rigidbodies, std::vector<byte>& hasRigidBodies,
```

Comment:
ps: `std::vector<bool>` get's turned into a dynamic bitflag behind the scenes. it's a special type of vector. each item takes up 1 bit. by using std::vector<byte> you're forcing it to be 8 bits. dunno if you meant to do that?

Reply:
I actually do recall that switching from std::vector<bool> to std::vector<byte> is something that you suggested, but I can't actually remember why, perhaps this can be investigated in another pr?

Reply:
yeah sure, that works for me. it just looks kindah strange to me and feels like a code smell. but if you wanna leave that for a different PR then that's fine by me